app/models/katello/concerns/organization_extensions.rb

```diff
@@ -37,10 +37,6 @@ module OrganizationExtensions
         scoped_search :on => :label, :complete_value => :true
 
         after_create :associate_default_capsule
-
-        validates :name, :uniqueness => true, :presence => true
-        validates_with Validators::KatelloNameFormatValidator, :attributes => :name
-        validates :label, :presence => true
         validates_with Validators::KatelloLabelFormatValidator, :attributes => :label
         validate :unique_name_and_label
```

Comment:
you're referring to the 3rd condition here:  https://github.com/Katello/katello/blob/master/app/models/katello/concerns/organization_extensions.rb#L53-L54   ?